docs/docs/api/06-Configuration/askui-ui-control-client.md

```diff
@@ -80,7 +80,7 @@ Setting credentials in code:
  let aui = await UiControlClient.build({
     credentials: {
       workspaceId: 'workspace_id',
-      token: 'userToken',
+      token: 'access token',
```

Comment:
Good catch!

Reply:
That is a good idea and a nice process improvement :)